mycroft/deprecated/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+This module contains dead code only
+All code here is in life support and available only for backwards api compatibility
+Bug fixes will be made to maintain compatibility but are very low priority!
+"""
```

Comment:
Consider logging a deprecation warning here to cover any deprecated imports

Reply:
deprecation warnings are already in relevant places, this is only documentation for users looking at the code

logging here would be super spammy since it may be imported in some places for backward compat